src/library/scala/collection/mutable/PriorityQueue.scala

```diff
@@ -142,9 +204,11 @@ sealed class PriorityQueue[A](implicit val ord: Ordering[A])
   def dequeue(): A =
     if (resarr.p_size0 > 1) {
       resarr.p_size0 = resarr.p_size0 - 1
-      resarr.p_swap(1, resarr.p_size0)
+      val elementToReturn = resarr.p_array(1)
```

Comment:
I think `result` is a more common name for temporary storage for a return value.